homeworks/yaroslav_sakhno/hw01/test.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import argparse
+from pyats.aetest import Testcase, test, main
+from calculation import add, divide
+
+
+class TryTest(Testcase):
+
+    parameters = {
+        'num1': 3,
+        'num2': 0
+    }
+
+    @test
+    def test_add(self,steps, num1, num2):
+        with steps.start('Add'):
+            assert add(num1,num2) == num1 + num2
+        with steps.start('Result > 0'):
+            try:
+                assert add(num1, num2) > 0
+            except AssertionError:
+                self.skipped('Result < 0)')
+
+    @test
+    def test_divide(self, steps, num1, num2):
+        with steps.start('Add'):
+            try:
+                assert divide(num1, num2) == num1 / num2
+            except ZeroDivisionError:
+                self.passx('passx')
+        with steps.start('Result > 0'):
+            try:
+                assert add(num1, num2) > 0
```

Comment:
Why you test "add" function in "test_divide"?

Reply:
@AlexRozl fixed

Reply:
@AlexRozl one more)